packages/network-controller/src/index.ts

```diff
@@ -1,4 +1,38 @@
-export * from './NetworkController';
+export type {
+  Block,
+  NetworkMetadata,
+  NetworkConfiguration,
+  BuiltInNetworkClientId,
+  CustomNetworkClientId,
+  NetworkClientId,
+  NetworksMetadata,
+  NetworkState,
+  BlockTrackerProxy,
+  ProviderProxy,
+  NetworkControllerStateChangeEvent,
+  NetworkControllerNetworkWillChangeEvent,
+  NetworkControllerNetworkDidChangeEvent,
+  NetworkControllerInfuraIsBlockedEvent,
+  NetworkControllerInfuraIsUnblockedEvent,
+  NetworkControllerEvents,
+  NetworkControllerGetStateAction,
+  NetworkControllerGetEthQueryAction,
+  NetworkControllerGetNetworkClientByIdAction,
+  NetworkControllerGetSelectedNetworkClientAction,
+  NetworkControllerGetEIP1559CompatibilityAction,
+  NetworkControllerFindNetworkClientIdByChainIdAction,
+  NetworkControllerSetProviderTypeAction,
+  NetworkControllerSetActiveNetworkAction,
+  NetworkControllerGetNetworkConfigurationByNetworkClientId,
+  NetworkControllerActions,
+  NetworkControllerMessenger,
+  NetworkControllerOptions,
+} from './NetworkController';
+export {
+  getDefaultNetworkControllerState,
+  knownKeysOf,
+  NetworkController,
```

Comment:
Could `RpcEndpointType` be exported too?  I have some places it would be useful

```suggestion
  NetworkController,
  RpcEndpointType
```

Reply:
Sure! Done.